google/cloud/pubsublite/BUILD

```diff
@@ -16,12 +16,18 @@ package(default_visibility = ["//visibility:private"])
 
 licenses(["notice"])  # Apache 2.0
 
-load(":google_cloud_cpp_pubsublite.bzl", "google_cloud_cpp_pubsublite_hdrs", "google_cloud_cpp_pubsublite_srcs")
+SOURCE_REGEX = "**/*.cc"
```

Comment:
nit: that is not a regex, but a glob expression or something?

Reply:
Done.